server.go

```diff
@@ -960,3 +975,69 @@ func (s *Server) ChannelFinalized(pid funding.PendingChanID) error {
 
 	return s.cfg.AuxFundingController.ChannelFinalized(pid)
 }
+
+// HandleTraffic is called in order to check if the channel identified by the
+// provided channel ID may have external mechanisms that would allow it to carry
+// out the payment.
+//
+// NOTE: This method is part of the routing.TlvTrafficShaper interface.
+func (s *Server) HandleTraffic(cid lnwire.ShortChannelID,
+	fundingBlob lfn.Option[tlv.Blob]) (bool, error) {
+
+	srvrLog.Debugf("HandleTraffic called, cid=%v, fundingBlob=%v", cid,
+		spew.Sdump(fundingBlob))
+
+	select {
+	case <-s.ready:
+	case <-s.quit:
+		return false, fmt.Errorf("tapd is shutting down")
+	}
+
+	return s.cfg.AuxTrafficShaper.HandleTraffic(cid, fundingBlob)
+}
+
+// PaymentBandwidth returns the available bandwidth for a custom channel decided
+// by the given channel aux blob and HTLC blob. A return value of 0 means there
+// is no bandwidth available. To find out if a channel is a custom channel that
+// should be handled by the traffic shaper, the HandleTraffic method should be
+// called first.
```

Comment:
The `HandleTraffic` method is basically a `CanHandle`, it was just named slightly differently. I guess it made sense from the point of view of where we use this interface in `lnd`:

```
		fundingBlob := link.FundingCustomBlob()
		shouldHandle, err := ts.HandleTraffic(cid, fundingBlob)
		if err != nil {
			return fmt.Errorf("traffic shaper failed to decide "+
				"whether to handle traffic: %w", err)
		}

		log.Debugf("ShortChannelID=%v: external traffic shaper is "+
			"handling traffic: %v", cid, shouldHandle)

```

I'm keeping the name for now (since it would require a new PR to `lnd` otherwise) but updated the description  a bit.